app/src/main/java/org/simple/clinic/editpatient/deletepatient/DeletePatientScreen.kt

```diff
@@ -56,8 +60,16 @@ class DeletePatientScreen(context: Context, attrs: AttributeSet) : ConstraintLay
 
     toolbar.setNavigationOnClickListener { screenRouter.pop() }
     with(deleteReasonsRecyclerView) {
+      adapter = deleteReasonsAdapter
       addItemDecoration(DividerItemDecorator(context, marginStart = 56.dp, marginEnd = 16.dp))
     }
+
+    deleteReasonsAdapter
+        .itemEvents
+        .takeUntil(detaches())
+        .ofType<DeleteReasonItem.Event.Clicked>()
+        .map { PatientDeleteReasonClicked(it.reason) }
+        .subscribe { events.onNext(it) }
```

Comment:
1. None of the events from `DeletePatientEvent` are getting pushed to analytics. Can we look at user generated events and ensure they are pushed correctly?
2. Can we follow our usual pattern of forwarding events?

```kotlin
val events: Observable<DeletePatientEvent> by unsafeLazy {
  Observable.mergeArray(
    dialogEvents(), // this can be the event subject that we have created
    adapterEvents() // set up another event stream here from the adapter
  )
  .compose(ReportAnalyticsEvents()) // This is missing
}
```